src/databricks/labs/blueprint/paths.py

```diff
@@ -351,13 +344,13 @@ def with_suffix(self, suffix):
             raise ValueError(msg)
         return self.with_name(stem + suffix)
 
-    def relative_to(self, other, *more_other, walk_up=False):  # pylint: disable=arguments-differ
-        other = self.with_segments(other, *more_other)
-        if self.anchor != other.anchor:
-            msg = f"{str(self)!r} and {str(other)!r} have different anchors"
+    def relative_to(self: P, *other: str | bytes | os.PathLike, walk_up: bool = False) -> P:  # pylint: disable=arguments-differ,useless-suppression
```

Comment:
remove useless suppression

Reply:
Locally I'm linting against all supported python versions. This is an example where the superclass method signature changed in 3.12. The suppression logic is: we need `arguments-differ` to avoid the error in 3.12, and `useless-suppression` to suppress the aforementioned in 3.10 and 3.11.

We have 3 options here:

1. Remove the type hints. (For some reason pylint only triggers if the type hints are present.)
2. Drop the suppressions as suggested. CI will pass, because it only checks against 3.10, but we know it will fail with 3.12.
3. Leave it as-is.

The failure with 3.12 is why I'm confirming what we want to do here; what is your preference?

Reply:
> Drop the suppressions as suggested

we lint against 3.10, because it's our primary python version at the moment.